libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -521,6 +529,10 @@ void AP_MotorsMulticopter::output_logic()
             } else if (_throttle_thrust_max < 0.0f) {
                 _throttle_thrust_max = 0.0f;
             }
+
+            // initialise motor failure variables
+            _thrust_boost = false;
+            _thrust_boost_ratio = 0.0f;
```

Comment:
You mean:
_thrust_boost_ratio = MAX(0.0, _thrust_boost_ratio-1.0f/(_spool_up_time*_loop_rate));
But yes that would make sense and we can do that although I am not sure if it can ever have an impact.

Reply:
Ok, I see what you are saying.

We should never be using boost in spool up or spool down so we should always be reducing it to zero. But I think you are right that we should be doing the same as in Spool Down to prevent a discontinuity.

Reply:
new commit added to fix this.  see "AP_Motors: handle boost in spool up and down the same way"